account_statement_import_paypal/models/account_statement_import_paypal_parser.py

```diff
@@ -207,18 +207,18 @@ def _convert_line_to_transactions(self, line):
             note += " (%s)" % payer_email
 
         unique_import_id = "{}-{}".format(transaction_id, int(timestamp.timestamp()))
-        name = (invoice or details or description or "",)
+        name = (invoice or transaction_id or "",)
         transaction = {
-            "name": invoice or details or description or "",
+            "name": name[0],
```

Comment:
I believe the best way to deal with the API change in Odoo 14, where the statement line inherits from account.move, the `name` attribute should not be set. Instead, the data that was assigned to `name` in earlier versions can be assigned to `ref` instead. See for instance this migration of a paypal module: https://github.com/OCA/bank-statement-import/pull/452/commits/12b7bfa128ac6d1fb313e529478b5e78a4393cf3#diff-84f1950cdde1a8ef9a098ed89c154cb4afd13d955a0088c978a570b7461ab604L295-R295

Do you think that would work in this case too?